Gems/Multiplayer/Code/Source/AutoGen/AutoComponent_Header.jinja

```diff
@@ -132,7 +132,7 @@ void {{ PropertyName }}({{ ', '.join(paramDefines) }});
 #}
 {% macro DeclareRpcInvocations(Component, Section, HandleOn, ProctectedSection) %}
 {% call(Property) AutoComponentMacros.ParseRemoteProcedures(Component, Section, HandleOn) %}
-{%    if Property.attrib['IsPublic']|booleanTrue == ProctectedSection %}
+{%    if Property.attrib['IsPublic']|booleanTrue != ProctectedSection %}
```

Comment:
We do != most everywhere else so this departure from convention caused permissions to get reversed.

Reply:
If it's not used in too many places might update Proctected -> Protected

Reply:
Ha! I didn't even notice the typo. I've changed this to IsProtected to match other functions.